dotcom-rendering/src/discussion-rendering/lib/api.tsx

```diff
@@ -248,6 +249,7 @@ export const recommend = (commentId: number): Promise<boolean> => {
 	}).then((resp) => resp.ok);
 };
 
+//todo: adjust this when the endpoint is ready
 export const addUserName = (userName: string): Promise<UserNameResponse> => {
 	const url = options.idApiUrl + `/user/me` + objAsParams(defaultParams);
```

Comment:
not related to this PR, but I think we don't need `defaultParams` when the root is `idApiUrl` -- I think they're only needed when we are calling Discussion API

Reply:
Yep, good spot, will adjust this in the discussion-rendering PR.